tests/lib/tools/suite/tests.session-support/task.yaml

```diff
@@ -1,5 +1,14 @@
 summary: check the session support status of various distributions
 execute: |
+
+    # In case ubuntu core is not already built in prepare helper, the check for uc18
+    # is the same than for ubuntu 18.04
+    if [[ "$SPREAD_SYSTEM" = ubuntu-core-18-* ]]; then
+        if MATCH '^ID=ubuntu$' < /etc/os-release; then
```

Comment:
nitpick: could we compress these two into one if?

Reply:
Sure, I'll do it in a following pr so we unblock the other pr first.
Thanks for hte review